datatypes/src/commonMain/kotlin/at/asitplus/crypto/datatypes/asn1/Asn1Encoder.kt

```diff
@@ -10,8 +10,6 @@ import at.asitplus.crypto.datatypes.asn1.BERTags.NULL
 import at.asitplus.crypto.datatypes.asn1.BERTags.UTC_TIME
 import at.asitplus.crypto.datatypes.asn1.DERTags.toExplicitTag
 import at.asitplus.crypto.datatypes.io.BitSet
-import io.matthewnelson.encoding.base16.Base16
-import io.matthewnelson.encoding.core.Encoder.Companion.encodeToString
 import kotlinx.datetime.Instant
 import kotlin.io.encoding.ExperimentalEncodingApi
```

Comment:
btw: Class name `Asn1TreeBuilder` does not match file name `Asn1Encoder`.
Also, the `@file:OptIn(ExperimentalEncodingApi::class)` seems to be unecessary?

Reply:
yup, the annotation was a remnant. gone now.

I renamed Asn1Encoder.kt → Asn1Encod*ing*.kt and Asn1Reader → Asn1Decoding.kt  for consistency, as both files contain more than just one toplevel class